expan/core/statistics.py

```diff
@@ -134,6 +134,37 @@ def sample_size(x):
     return len(x) - x_nan
 
 
+def estimate_sample_size(x, mde, n, r, alpha=0.05, beta=0.2):
+    """
+    Estimates sample size based on sample mean and variance given MDE (Minimum Detectable effect), number of variants and variant split ratio
+
+    Args:
+        x (pd.Series or pd.DataFrame): sample to base estimation on
```

Comment:
just curious - why not `numpy.array`?

Reply:
could be an array, there's a nice benefit of allowing it to be a DataFrame since we can calculate the estimate for multiple columns simultaneously

Reply:
@mkolarek thanks, I see!